packages/bpk-docs/src/components/DocsPageBuilder/NeoDocsPageBuilder.js

```diff
@@ -195,15 +195,15 @@ const NeoDocsPageBuilder = props => {
 
   return (
     <BpkContentContainer
-      className={getClassName(
-        `bpkdocs-content-page--${
-          sections.length % 2 === 0 ? 'even' : 'odd'
-        }-sections`,
-      )}
```

Comment:
This code didn't appear to be having any effect, so I've replaced it with logic to change the background colour if the page is wrapped.

Reply:
@k0nserv might have more insight?

Reply:
It's so that the background has the same color as the last section.